docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -69,27 +67,25 @@ If {ProductName} can’t access your Git repository, see <<Granting App Studio a
 |N/A
 |===
 
-//// 
-From JSese: May 15, 2023, Commenting out per Matt Reid and Christian V. Support for private repos is post-Summit.
 [discrete]
-== Granting App Studio access to your Git repository
+[#importing-code-from-private-repo]
+=== Importing code from a private GitHub repository
 
-If {ProductName} can’t access your Git repository, the following message displays: “We can’t access your repository, probably because it’s either private or the URL is incorrect.” If your repository is private, choose one of the following two ways to allow {ProductName} to interact with your repository:
+To use your code from a private repository to build your application, complete the following steps:
 
-[discrete]
-=== Granting App Studio access to your Git repository through GitHub OAuth
-
-. Under **Authorization**, click **Sign in**. **GitHub Authorize OAuth** opens in a new tab.
-. Click **Authorize _redhat-appstudio_**.
-
-[discrete]
-=== Granting App Studio access to your Git repository with a token
+. Enter the link to your private repository in the **Git repository URL** field. {ProductName} displays the following message: “We can’t access your repository, probably because it’s either private or the URL is incorrect.” 
+. Grant {ProductName} access to your repository.
+.. On the **Grab some code** page, enter the URL to your repository. After {ProductName} detects that it needs access, the **Authorization** section opens.
```

Comment:
I'm not sure if these need to be substeps, In fact ,it might read easier to simply be 1. 2. 3. 4. 

However, it's not incorrect and doesn't read badly, so if you want to keep it substeps, then you can!

Reply:
oooh, nice idea! lemme look!

Reply:
ok, took a look, and i feel like the "Grant access to your [repo]..." step could almost stand alone as its own procedure, so i feel like substeps make more sense. i'm glad you're ok with either bc your input is invaluable. thank you, @jc-berger!